docs/OPS_RUNBOOK.md

```
@@ -223,10 +226,10 @@ du -xhd1 /home/corgi | sort -h
 ```bash
 sudo /usr/local/bin/bluesky-ops-retention.sh
 ```
-3. If backup directory is large, keep latest 5 only:
+3. Verify the backup directory contains only the latest 5 valid dumps:
```

markdownlint-cli2 (0.22.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/1/1

(MD029, ol-prefix)